common/models/user.js

```diff
@@ -340,7 +341,33 @@ module.exports = function(User) {
             new Error(`${email} is already associated with another account.`)
           );
         }
-        return this.update$({ email }).toPromise();
+
+        const emailVerified = false;
+        return this.update$({
+          email, emailVerified
+        })
+        .do(() => {
+          this.email = email;
+          this.emailVerified = emailVerified;
+        })
+        .flatMap(() => {
+          var mailOptions = {
+            type: 'email',
+            to: email,
+            from: 'Team@freecodecamp.com',
+            subject: 'Welcome to Free Code Camp!',
+            template: path.join(
+              __dirname,
+              '..',
+              '..',
+              'server',
+              'views',
+              'emails',
+              'user-email-verify.ejs'
+            )
+          };
+          return Observable.fromPromise(this.verify(mailOptions));
```

Comment:
Todo : CHANGE THIS, because loopback will return the data that we do not want to be sent back over the AJAX.